lib/rouge/lexers/email.rb

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*- #
+# frozen_string_literal: true
+
+module Rouge
+  module Lexers
+    class Email < RegexLexer
+      tag 'email'
+      aliases 'eml', 'e-mail'
+      filenames '*.eml'
```

Comment:
@smokris You had `*.mail` as a file extension that should be recognised as being e-mail. I didn't look very hard but it appeared as if `*eml` is the standard extension (and possibly `*.msg` if you want to include Outlook's non-standard way of doing things).

Reply:
Using just `*.eml` is fine with me.